src/app/modules/shared/components/technologies/technologies.component.ts

```diff
@@ -17,7 +17,7 @@ export class TechnologiesComponent implements OnInit, OnDestroy {
   readonly ALLOW_SELECT_MULTIPLE = true;
   readonly ALLOW_SEARCH = true;
   readonly MIN_SEARCH_TERM_LENGTH = 2;
-  readonly TYPE_TO_SEARCH_TEXT = 'Please enter 2 or more characters';
+  readonly TYPE_TO_SEARCH_TEXT = 'Please enter 2 or more characters to search for technologies ';
```

Comment:
It is better is rename to TYPE_TO_SEARCH_TECHNOLOGIES, because it cannot be used for other search term